crates/nu-command/src/filesystem/touch.rs

```diff
@@ -244,6 +244,11 @@ impl Command for Touch {
                 example: r#"touch -m -r fixture.json d e"#,
                 result: None,
             },
+            Example {
+                description: r#"Update times of all JSON files in folder"#,
+                example: r#"touch ...(glob *.json)"#,
```

Comment:
```suggestion
                example: r#"touch ...(ls *.json).name"#,
```

Even though `touch` won't be around for much longer, I think we should generally avoid suggesting the usage of the `glob` command, except for the specific globbing features that `glob` command supports which nu-glob does not (like brace expansions). It's my understanding that when the additional globbing features are added into nu-glob, the `glob` command will be deprecated and `ls` will be used instead, so I think we should point users towards `ls`.

Reply:
Oh, I didn't know that we'd be getting rid of `glob` in the future, thanks